actors/verifreg/tests/verifreg_actor_test.rs

```diff
@@ -790,76 +786,4 @@ mod datacap {
         }
         h.check_state(&rt);
     }
-
-    #[test]
-    fn restore() {
```

Comment:
I have a feeling we'll want to reinstate these tests for verifreg::RemoveExpiredAllocations.  A follow up is fine but in that case maybe leave these tests commented out so review is easier?

Reply:
I looked through and I think they'll be different enough to need basically re-writing. It's a good point to come check here for reference though.

Reply:
Looking through these now while finishing off testing:
- We use actor IDs for the client/provider, so no longer need to resolve addresses
- Anyone can call RemoveExpiredAllocations, not just the market actor
- No need to check minimums since we're dealing with allocation records the verifreg already has